zest/releaser/tests/utils.txt

```diff
@@ -458,8 +458,9 @@ We want to discover errors and show them in red.
     '\x1b[31m'
     >>> Fore.MAGENTA
     '\x1b[35m'
-    >>> utils.execute_command(['ls', 'some-non-existing-file'])
-    '\x1b[31mls...'
+    >>> output = utils.execute_command(['ls', 'some-non-existing-file'])
+    >>> output.startswith(Fore.RED)
```

Comment:
The output somehow started with double quotes, now, instead of single quotes.
That's why I changed the testing style.